operations/README.md

```diff
@@ -82,6 +82,12 @@ This section describes all operations supported by the API, organised here by th
 | [Add alternative payment](payments.md#add-alternative-payment) | Adds a new alternative payment to a specified customer |
 | [Add outlet bills](outletbills.md#add-outlet-bills) | Adds new outlet bills with their items |
 
+## Loyalty Programs
```

Comment:
This is implemented fine, but can we just confirm that a new section is really needed for Loyalty Programs, with a single endpoint? Will there be more endpoints added in future, or is this something that could be fitted under the Enterprise heading?